app/ts/ReportingUI/components/Reporting.tsx

```diff
@@ -24,32 +21,87 @@ export const DisplayExtraInfo = ({ marketData }: MarketProps) => {
 	if (marketData.deepValue === undefined) return <></>
 	if (marketData.deepValue.parsedExtraInfo === undefined) {
 		return <>
-			<p> Failed to parse Extra data, unparsed extra data: </p>
+			<span> Failed to parse Extra data, unparsed extra data: </span>
```

Comment:
Style: I'm not personally a fan of spaces around HTML elements.  I know that they are *usually* stripped, but I don't think they are *always* stripped (e.g., `<pre>` and `<code>` elements) and I prefer consistency so you don't have to know exactly which elements need stripping and which ones don't.

Same comment for bunch of this document, recommend removing spaces between elements, or having a discussion on the benefits vs costs.

```suggestion
			<span>Failed to parse Extra data, unparsed extra data:</span>
```